datahub-web-react/src/app/home/AcrylDemoBanner.tsx

```diff
@@ -0,0 +1,56 @@
+import Link from 'antd/lib/typography/Link';
+import React from 'react';
+import styled from 'styled-components';
+import AcrylLogo from '../../images/acryl-light-mark.svg';
+
+const BannerWrapper = styled.div`
+    padding: 12px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #262626;
+    background-color: #e6f4ff;
```

Comment:
May be nice to put these into a constants, but I bet they're only used here for now. If so, it's fine the way it is.